doc/sphinx/scripting/scripting-alpha.rst

```diff
@@ -1813,6 +1813,9 @@ the fourth argument you must specify the second and third arguments too.
      what the fstype setting is.
    * 4 => load all glyphs from the 'glyf' table of a ttc font (rather than only
      the glyphs used in the font picked).
+   * 8 => Report more error conditions
+   * 16 => Do not create a view window for this font even if the UI is active. Note: This option supports efficient bulk processing of fonts in scripts run through the UI but using it can be tricky. Open fonts will be listed at the bottom of the "Window" menu but choosing them will have no effect. If some fonts are not closed you may need to "force-quit" the application using your OS.
```

Comment:
Ah, I guess we might as well wrap this too (and the other lines)

Reply:
Half the time I've tried to wrap the lines in these lists it has worked fine and the other half of the time it has failed and I don't know the difference. Could you please do the commit for this one? You have access to the branch via github.

Reply:
Sure.

For reference, the thing to keep in mind with lists is that the indentation has to line up like so:

```rst

* test
  note this is indented two spaces to line up with the line above

1. numbered
   this is indented by 3 spaces to line up with above

.. object:: stuff

   this is indented 3 spaces

   * stuff
     indented 5 spaces
```